src/scss/attendee/variables.scss

```diff
@@ -14,6 +16,15 @@ $button-height: $whitespace * 3.5;
 $blur: 5px;
 $icon-size: 1.3rem;
 
+$element-height-normal: 42px;
+
+$border-radius-normal: 4px;
+
+$font-size-large: 23px;
+
+$z-index-attendee-overlay: 1;
```

Comment:
I could make a simple PR for a `layer.scss` or similarly named file in order to have something like the suggestion here https://doesitsparkle.slack.com/archives/C031U4S7D08/p1645090984517579

Reply:
++ let's do it